docs/sphinx/source/fsd/index.md

```diff
@@ -0,0 +1,64 @@
+
+### History
+
+The idea for what became the Open Navigation Surface Project had been discussed within the US hydrographic community for some time, but first crystallized in late 2003 through comments of M. Paton at IVS3D Ltd., which were adopted by B. Calder at CCOM/JHC with the intent of acting as an independent third-party broker for the development of the library. The ONSWG first met in early 2004, and outlined the requirements for the BAG file structure, the ethos of the project and the basic functionality of the source code library.  A presentation on the structure of the project was made at US Hydro 2005 in March, and development continued until the second ONSWG meeting in mid 2005, when the majority of the first release of the library was pulled together in a little under a week.  A second presentation, including demonstrations of the library linked into SAIC SABER, CARIS HIPS and IVS Fledermaus, was given at Shallow Survey 2005 in September and development of the library continued using e-mail and telephone conferencing until the first Candidate Release on 8 February 2006. Following comments from users and further development, the first official release of the library was made on 8 April 2006.
+
+### Identification
+
+This document describes the Bathymetric Attributed Grid file format, version 2.0.1.  This version corresponds to the Git release tag “release-2.0.1”, which may be obtained from the Open Navigation Surface (ONS) Project on [GitHub](https://github.com/OpenNavigationSurface/BAG).
+
+This release of the library supports, and has been tested on 64-bit versions of Linux, Windows 10, and macOS. The source code is written in C++14 and builds with CMake, and should therefore compile on most Unix-like systems if required.  Users who successfully compile the library on other platforms are urged to provide information on any modifications required to the project so that these platforms can be supported in future releases.
```

Comment:
This paragraph is a bit contradictory when compared to the statement that the "document describes the Bathymetric Attributed Grid file format, version 2.0.1". Is this document about a file format or about a library?

Reply:
I have removed the second paragraph.